packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -421,8 +421,15 @@ export default {
 
             state.time = fulfilment.time;
 
-            if (fulfilment.location && fulfilment.location.address && fulfilment.location.address.lines) {
-                const { address } = fulfilment.location;
+            let address = null;
+            if (addressService.isAddressInLocalStorage()) {
+                address = addressService.getAddressFromLocalStorage();
+            } else if (fulfilment.location && fulfilment.location.address && fulfilment.location.address.lines) {
+                /* eslint-disable prefer-destructuring */
```

Comment:
Is this bit still needed? I imagine it won't ask you to destructure it now its declared on a different line; could be wrong though